pkg/stub/imagestreams.go

```diff
@@ -186,7 +186,7 @@ func (h *Handler) upsertImageStream(imagestreamInOperatorImage, imagestreamInClu
 		return nil
 	}
 
-	h.updateDockerPullSpec([]string{"docker.io", "registry.redhat.io", "registry.access.redhat.com", "quay.io"}, imagestreamInOperatorImage, opcfg)
+	h.updateDockerPullSpec([]string{"docker.io", "registry.access.redhat.com", "quay.io"}, imagestreamInOperatorImage, opcfg)
```

Comment:
i don't think this should be changing?

Reply:
this was suppose to be an optimization with the default being registry.redhat.io now but you are right / I forgot if they override the registry we want to replace those too

that array string is things we want to replace